src/components/dashboard/dashboard.spec.ts

```diff
@@ -0,0 +1,277 @@
+                                        <h4 class="dashboard-run-info-subtitle">16:32 14min</h4>
+                                    </div>
+
+                                </div>
+
+                                <div class="dashboard-widget-content stacked">
+                                    <div class="dashboard-run-info">
+                                        <h3 class="dashboard-run-info-title">NV Test_22</h3>
+                                        <h4 class="dashboard-run-info-subtitle">ID Test Name</h4>
+                                    </div>
+
+                                    <div class="dashboard-run-info">
+                                        <h3 class="dashboard-run-info-title">PC_hiLA</h3>
+                                        <h4 class="dashboard-run-info-subtitle">Test Name</h4>
+                                    </div>
+
+                                    <div class="dashboard-run-info muted">
+                                        <h3 class="dashboard-run-info-title">16/01/2017</h3>
+                                        <h4 class="dashboard-run-info-subtitle">16:32 14min</h4>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </ux-dashboard-widget>
+
+                </ux-dashboard>
+    `
+})
+export class DashboardComponent {
+
+    options: DashboardOptions;
+    initialOptions: DashboardOptions  = {
+        columns: 3,
+        padding: 10,
+        rowHeight: 200,
+        emptyRow: false,
+        minWidth: 187
+    };
+}
+
+describe('Dashboard', () => {
+    let component: DashboardComponent;
+    let fixture: ComponentFixture<DashboardComponent>;
+    let nativeElement: HTMLLIElement;
+    let widget: HTMLLIElement;
+    let widgetC: HTMLElement;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            imports: [DashboardModule],
+            declarations: [DashboardComponent]
+        })
+            .compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(DashboardComponent);
+        component = fixture.componentInstance;
+        nativeElement = fixture.nativeElement;
+        widget = nativeElement.querySelector('ux-dashboard-widget');
+        widgetC = nativeElement.querySelector('#widget-C');
+        fixture.detectChanges();
+    });
+
+    it('should initialise correctly', () => {
+        expect(component).toBeTruthy();
+    });
+
+    it('should reposition widgets if columns option adjusted', async() => {
+        let adjustedOptions: DashboardOptions  = {
+            columns: 3,
+            padding: 5,
+            rowHeight: 400,
+            emptyRow: false,
+            minWidth: 187
+        };
+        component.options = component.initialOptions;
+        fixture.detectChanges();
+        await fixture.whenStable();
```

Comment:
As noted above, these 3 lines can be removed if the test component is set up correctly.